gcp/appengine/frontend_handlers.py

```diff
@@ -748,14 +748,17 @@ def link_to_deps_dev(package, ecosystem):
 def display_severity_rating(severity: dict) -> str:
   """Return base score and rating of the severity."""
   severity_base_score, severity_rating = calculate_severity_details(severity)
+  if severity_base_score is None:
+    return "Invalid Severity Rating"
+
   return f"{severity_base_score} ({severity_rating})"
 
 
 @blueprint.app_template_filter('severity_level')
 def severity_level(severity: dict) -> str:
   """Return rating of the severity."""
   _, rating = calculate_severity_details(severity)
-  return rating.lower()
+  return (rating and rating.lower()) or "invalid"
```

Comment:
nit: double quotes

Also, imo this might be a bit clearer:
```suggestion
  return 'invalid' if rating is None else rating.lower()
```

Reply:
Yeah good idea!